lite/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM alpine:3.15
+ENV LANG C.UTF-8
+
+RUN apk update
+RUN apk upgrade
+RUN apk add \
+        clang \
+        clang-extra-tools \
+        clang-analyzer \
+        lld \
+        cmake \
+        make \
+        doxygen \
+        git \
+        python3 \
+        py3-pip \
+        bash \
+        gcc-arm-none-eabi \
+        newlib-arm-none-eabi \
+        protoc
```

Comment:
Please sort package names [alphabetically](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments).

Reply:
Done in [fe172af](https://github.com/LedgerHQ/ledger-app-builder/pull/36/commits/fe172afb8eadcb3519017aec39cf3e8784bb0101)